src/__tests__/App.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+// import renderer from "react-test-renderer";
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes. This is just one line and it is way too small to give you a `change request` for, So it would be very nice to remove it before you merge this PR.